index.js

```diff
@@ -1,4 +1,20 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const { addProduct, getProducts } = require('./controllers/productsControllers');
+
+const app = express();
+const PORT = 3000;
+const STATUS_OK = 200;
+
+app.use(bodyParser.json);
```

Comment:
`bodyParser.json` é uma função que retorna o middleware que transforma o corpo da requisição na chave `body` que a gente conhece no _express_
```suggestion
app.use(bodyParser.json);
```